### PR TITLE
Version 2

### PR DESCRIPTION
**Pull request by ericgaspar**
## Problem

- *Description of why you made this PR*

## Solution

- *And how do you fix that problem*

## PR Status

- [ ] Code finished and ready to be reviewed/tested
- [ ] The fix/enhancement were manually tested (if applicable)

## Automatic tests

Automatic tests can be triggered on https://ci-apps-dev.yunohost.org/ *after creating the PR*, by commenting "!testme", "!gogogadgetoci" or "By the power of systemd, I invoke The Great App CI to test this Pull Request!". (N.B. : for this to work you need to be a member of the Yunohost-Apps organization)

### CODE REVIEW

**Comment by ericgaspar** [comment]
!testme

**Comment by ericgaspar** [comment]
!testme

**Comment by alexAubin** [comment]
!testme

File: manifest.toml [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+website = "https://grafana.com/"
+demo = "https://play.grafana.org"
+code = "https://github.com/grafana/grafana"
+
+[integration]
+yunohost = ">= 11.2"
+architectures = "all"
+multi_instance = false
+ldap = false
+sso = false
```
Comment (alexAubin) [inline manifest.toml]: The ADMIN.md says ldap and sso is supported but stuff is set to false here ...

File: doc/ADMIN.md
Comment (alexAubin) [inline doc/ADMIN.md:1]: To be split into proper PRE_INSTALL, POST_INSTALL, DESCRIPTION.md ...

File: conf/grafana.ini [hunk 2]
```diff
@@ -0,0 +1,1582 @@
...
+#reset_basic_roles = false
+
+# Validate permissions' action and scope on role creation and update
+; permission_validation_enabled = true
+
+#################################### SMTP / Emailing ##########################
+[smtp]
+enabled = true
+host = localhost:25
+;user = __APP__
```
Comment (alexAubin) [inline conf/grafana.ini:859]: Sounds like this shouldnt be commented ?

**Comment by ericgaspar** [comment]
!testme

**Pull request merged**